src/code/z_camera_data.c

```diff
@@ -0,0 +1,3775 @@
+    { 0x03FFFFFF, 0x00000005, sCamSetBossTwinmoldModes },          // CAM_SET_BOSS_TWINMOLD
+    { 0x03FFFFFF, 0x00000005, sCamSetBossGohtModes },              // CAM_SET_BOSS_GOHT
+    { 0x05FFEFF7, 0x00000005, sCamSetBossGyorgModes },             // CAM_SET_BOSS_GYORG
+    { 0x00000001, 0x00000005, sCamSetConnect0Modes },              // CAM_SET_CONNECT0
+    { 0x05FFEFF7, 0x00000005, sCamSetPinnacleRockModes },          // CAM_SET_PINNACLE_ROCK
+    { 0x03FFFFFF, 0x00000005, sCamSetNormal2Modes },               // CAM_SET_NORMAL2
+    { 0x03FFFFFF, 0x00000005, sCamSetBombBowlModes },              // CAM_SET_HONEY_AND_DARLING_2
+    { 0x00000001, 0x80000005, sCamSetCircAModes },                 // CAM_SET_CIRCLEA
+    { 0x05FFEFF7, 0x00000005, sCamSetWhirlpoolModes },             // CAM_SET_WHIRLPOOL
+    { 0x01FFFFDF, 0x00000005, sCamSetCuccoShackModes },            // CAM_SET_CUCCO_SHACK
+    { 0x03FFFFFF, 0x00000005, sCamSetGiantModes },                 // CAM_SET_GIANT
+    { 0x00000001, 0x00000008, sCamSetScene0Modes },                // CAM_SET_SCENE0
+    { 0x03FFFFFF, 0x00000005, sCamSetRoom1Modes },                 // CAM_SET_ROOM1
+    { 0x05FFEFF7, 0x00000005, sCamSetWater2Modes },                // CAM_SET_WATER2
+    { 0x03FFFFFF, 0x00000005, sCamSetWoodfallSwampModes },         // CAM_SET_WOODFALL_SWAMP
+    { 0x03FFFFFF, 0x00000005, sCamSetForceKeepModes },             // CAM_SET_FORCEKEEP
+    { 0x03FFFFFF, 0x80000005, sCamSetParallel1Modes },             // CAM_SET_PARALLEL1
+    { 0x00000001, 0xC0000008, sCamSetStart1Modes },                // CAM_SET_START1
+    { 0x03FFFFFF, 0x00000005, sCamSetRoom2Modes },                 // CAM_SET_ROOM2
+    { 0x03FFFFFF, 0x00000005, sCamSetNorm4Modes },                 // CAM_SET_NORMAL4
+    { 0x0000002F, 0x00000005, sCamSetElegyShellModes },            // CAM_SET_ELEGY_SHELL
+    { 0x03FFFFFF, 0x00000005, sCamSetDungeon4Modes },              // CAM_SET_DUNGEON4
+};
+
+static s16 sGlobalCamDataSettings[26] = {
+    /* -25 */ CAM_SET_ELEGY_SHELL,
+    /* -24 */ CAM_SET_SIDED,
+    /* -23 */ CAM_SET_BOAT_CRUISE,
+    /* -22 */ CAM_SET_NONE,
+    /* -21 */ CAM_SET_SUBJECTD,
+    /* -20 */ CAM_SET_NORMALD,
+    /* -19 */ CAM_SET_NONE,
+    /* -18 */ CAM_SET_NONE,
+    /* -17 */ CAM_SET_NONE,
+    /* -16 */ CAM_SET_WARP_PAD_ENTRANCE,
+    /* -15 */ CAM_SET_ATTENTION,
+    /* -14 */ CAM_SET_CONNECT0,
+    /* -13 */ CAM_SET_REMOTEBOMB,
+    /* -12 */ CAM_SET_NONE,
+    /* -11 */ CAM_SET_MASK_TRANSFORMATION,
+    /* -10 */ CAM_SET_LONG_CHEST_OPENING,
+    /*  -9 */ CAM_SET_REBIRTH,
+    /*  -8 */ CAM_SET_DEATH,
+    /*  -7 */ CAM_SET_WARP_PAD_MOON,
+    /*  -6 */ CAM_SET_NAVI,
+    /*  -5 */ CAM_SET_ITEM3,
+    /*  -4 */ CAM_SET_ITEM2,
+    /*  -3 */ CAM_SET_ITEM1,
+    /*  -2 */ CAM_SET_ITEM0,
+    /*  -1 */ CAM_SET_STOP0,
+    /*   0 */ CAM_SET_NONE,
+};
+
+static s16* sGlobalCamDataSettingsPtr = &sGlobalCamDataSettings[25];
```

Comment:
This seems to point to the last element, what about using an ARRAY_COUNT-1?
```suggestion
static s16* sGlobalCamDataSettingsPtr = &sGlobalCamDataSettings[ARRAY_COUNT(sGlobalCamDataSettings)];
```